src/pages/NotFound.jsx

```diff
@@ -11,18 +12,20 @@ const NotFound = () => {
   }, [])
 
   return (
-    <div className="flex min-h-screen flex-col items-center justify-center gap-4 bg-gray-100 p-16 text-center">
-      <h1 className="text-6xl font-bold text-primary-600">
-        404 - Page Not Found
-      </h1>
-      <p className="text-xl">
-        The page you were looking for does not exist, you will be redirected to
-        the home page in 5 seconds or you can click the link below.
-      </p>
-      <NavLink className="text-blue-500 hover:underline" to="/">
-        Return to Home Page
-      </NavLink>
-    </div>
+    <PageLayout>
```

Comment:
I think wrapping this in page layout is good and keeps things consistent

If we're adding now a Navbar here, we should probably do some conditional rendering like @JustinBeaudry is doing in #30 as most of the nav is tailored for the home page

A bit of a moot point as the user will get redirected there